app/src/main/java/com/practicum/playlistmaker/AudioPlayerActivity.kt

```diff
@@ -11,6 +15,31 @@ import java.text.SimpleDateFormat
 import java.util.*
 
 class AudioPlayerActivity : AppCompatActivity() {
+    companion object {
+        private const val STATE_DEFAULT = 0
+        private const val STATE_PREPARED = 1
+        private const val STATE_PLAYING = 2
+        private const val STATE_PAUSED = 3
+
+        private const val UPDATE_TIME_INFO = 500L
+    }
+
+    private var playerState = STATE_DEFAULT
+
+    private var mediaPlayer = MediaPlayer()
```

Comment:
```suggestion
    private val mediaPlayer = MediaPlayer()
```
Плеер не пересоздаётся в рамках активити, поэтому можно сделать val